Prediction.cs

```diff
@@ -432,7 +235,7 @@ internal static PredictionOutput GetPrediction(PredictionInput input, bool ft, b
             if (ft)
             {
                 //Increase the delay due to the latency and server tick:
-                input.Delay += Game.Ping / 2000f + 0.06f;
+                input.Delay += Game.Ping / 2000f + 0.05f;
```

Comment:
I would be pleased to show you the calculations. Will prepare a simple essay for you!